README.md

```diff
@@ -4,7 +4,7 @@
 
 **https://js.arcgis.com/next**
 
-Welcome to the next version of the ArcGIS API for JavaScript (4.24). This version of the API is currently in development and will be updated frequently until the production release in July 2022. Try out new features, perform regression testing, and get a feel for the next release - then share your feedback with the development team. Please make sure that bugs and enhancement requests are specific to https://js.arcgis.com/next and not applicable to a released version.
+Welcome to the next version of the ArcGIS API for JavaScript (4.25). This version of the API is currently in development and will be updated frequently until the production release in October 2022. Try out new features, perform regression testing, and get a feel for the next release - then share your feedback with the development team. Please make sure that bugs and enhancement requests are specific to https://js.arcgis.com/next and not applicable to a released version.
```

Comment:
I've been meaning to update this sentence to add the ES modules, here's my suggestion:

"Please make sure that bugs and enhancement requests are specific to https://js.arcgis.com/next or `@arcgis/core@next` and not applicable to a released version."

Reply:
Thanks @andygup ! Added 👍